docs/msw/install.txt

```diff
@@ -497,35 +497,43 @@ NB: The makefiles and project files provided with wxWidgets samples show which
     simply copy a makefile or project file from %WXWIN%\samples\minimal or some
     other sample and adapt it to your application.
 
-Independently of the compiler and make/IDE you are using you must do the
-following to use wxWidgets sources under the directory $WXWIN (notice that
-different tool chains refer to environment variables such as WXWIN in
-different ways, e.g. MSVC users should use $(WXWIN) instead of just $WXWIN):
+The instructions below suppose that wxWidgets sources are under the directory
+$WXWIN (notice that different tool chains refer to environment variables such
+as WXWIN in different ways, e.g. MSVC users should use $(WXWIN) instead of just
+$WXWIN). They also use $WXLIB as a shortcut for the subdirectory of $WXWIN\lib
+which is composed from several parts separated by underscore: first, a
+compiler-specific prefix (e.g. "vc" for MSVC, "gcc" for g++ or the value of
+COMPILER_PREFIX if you set it explicitly), then optional "x64" if building in
+64 bits and finally either "lib" or "dll" depending on whether static or
+dynamic wx libraries are being used. For example, it could be
+"$WXWIN\lib\vc_x64_lib" for 64-bit static libraries built with MSVC.
+
+The instructions themselves consist of the following steps:
 
 * Add $WXWIN\include to the
   - compiler
   - resource compiler
   include paths.
+* If using MSVC, prepend $WXWIN\include\msvc to the include paths too.
+  Otherwise, append $WXLIB\mswu[d] to the include paths, where "d" should
```

Comment:
Perhaps it is just me but perhaps using `$WXLIB`  like this may be a bit confusing?

`$WXLIB` is being used in the same context as `$WXIN`, however, the latter is an actual compiler/system environment variable which can be used in the IDE settings while the former is only a symbol used in the instructions.

Reply:
Yes, I agree that `$WXLIB` is confusing, but I don't know what else to use... `<wx-lib-dir>` perhaps?

Probably a good idea to add a link to http://docs.wxwidgets.org/trunk/overview_cmake.html#cmake_apps indeed...

Reply:
I think that using  `<wx-lib-dir>`, i.e., not using the same "syntax" as for `$WXWIN`, would be better as it is clear that it is only a placeholder and not an actual variable.

Regarding the CMake note, I meant placing it somewhere at the top of the whole file and link the whole overview, not referencing it only from the application-building part.